### PR TITLE
Added script for creating .zip file.

### PR DESCRIPTION
**Pull request by VaclavStupka**

### CODE REVIEW

File: src/CreatePackage.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        }
+    }
+
+    if (fs.existsSync('./dist')) {
+        options.typeScriptPackage = true;
+    }
+
+    try {
+        fs.unlinkSync(zipFile);
+    } catch {}
```
Comment (CamStreamer) [inline src/CreatePackage.ts:57]: Asi bych tu vyjimku netutlal..Vypsal bych ji a ukoncil skript. Muze tam byt zip na ktery nebudes mit prava na smazani napriklad..

File: src/CreatePackage.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    const zip: AdmZip = new AdmZip();
+    createZipArchive(zip, folder, options);
+    zip.writeZip(zipFile);
+}
+
+console.clear();
```
Comment (CamStreamer) [inline src/CreatePackage.ts:64]: Je vhodne volat clear? Nekdo to muze volat napriklad uprostred skriptu.. Nesmaze mu to pak predchozi vystup?

**Review by CamStreamer — CHANGES_REQUESTED** [review]

File: src/CreatePackage.ts (hunk 2 above)
Comment (VaclavStupka) [reply src/CreatePackage.ts:64]: !

File: src/CreatePackage.ts (hunk 1 above)
Comment (VaclavStupka) [reply src/CreatePackage.ts]: !

File: src/CreatePackage.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        if (arg == '-i' || arg == '-includeNodeModules') {
+            options.includeNodeModules = true;
+        }
+    }
+
+    if (fs.existsSync('./dist')) {
+        options.typeScriptPackage = true;
+    }
+
+    if (fs.existsSync('./' + zipFile)) {
```
Comment (CamStreamer) [inline src/CreatePackage.ts:55]: podle me neni potreba u relativnich cest davat ./
to same u dist vyse

File: src/CreatePackage.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        try {
+            fs.unlinkSync(zipFile);
+        } catch (error) {
+            console.log('An error occured: ', error);
```
Comment (CamStreamer) [inline src/CreatePackage.ts:59]: tady bych dal process.exit s nenulovym navratovym kodem

File: README.md [hunk 5]
```diff
@@ -334,3 +334,17 @@ np
 ```
 The utility will ask you for the prefered version raise and if you are lucky, everything will run just fine.
 Finally, edit GitHub release form.
+
+### Preparing a package to upload to CamScripter
+If you  want to create your own package and upload it to CamScripter, you can use script CreatePackage. It creates zip file which contains all required files and directories in your package folder. The script accepts source code written either in JavaScript or TypeScript if the package has correct structure (more information in https://github.com/CamStreamer/CamScripterApp_examples/#readme). To include this script into your package add in file `package.json` following lines:
```
Comment (CamStreamer) [inline README.md:339]: If you want to create your own package and upload it to CamScripter App, you can use the script CreatePackage. It creates a zip file which contains all required files and directories in your package folder. The script accepts source code written either in JavaScript or TypeScript if the package has the correct structure (more information in https://github.com/CamStreamer/CamScripterApp_examples/#readme). To include this script in your package add the following lines in the file `package.json`:

File: README.md [hunk 6]
```diff
@@ -334,3 +334,17 @@ np
...
+```json
+"scripts": {
+    "create-package": "node CamStreamerLib/CreatePackage.js"
+  }
+```
+By default the zipped package does not contain node_modules directory. If you want to include it (which is required e.g. when uploading to CamScripter), add this:
```
Comment (CamStreamer) [inline README.md:345]: By default, the zipped package does not contain node_modules directory. If you want to include it (required when uploading to CamScripter App on Axis camera), add this:

**Review by CamStreamer — CHANGES_REQUESTED** [review]

File: src/CreatePackage.ts (hunk 3 above)
Comment (VaclavStupka) [reply src/CreatePackage.ts:55]: !

File: README.md (hunk 6 above)
Comment (VaclavStupka) [reply README.md:345]: !

File: README.md (hunk 5 above)
Comment (VaclavStupka) [reply README.md:339]: !

File: src/CreatePackage.ts (hunk 4 above)
Comment (VaclavStupka) [reply src/CreatePackage.ts:59]: !

**Review by CamStreamer — APPROVED** [review]

**Pull request merged**